embassy-nrf/src/pdm.rs

```diff
@@ -145,15 +146,14 @@ impl<'d, T: Instance> Pdm<'d, T> {
     }
 
     fn _set_gain(r: &crate::pac::pdm::RegisterBlock, gain_left: I7F1, gain_right: I7F1) {
-        let gain_left = gain_left
-            .saturating_add(I7F1::from_bits(40))
-            .saturating_to_num::<u8>()
-            .clamp(0, 0x50);
-        let gain_right = gain_right
-            .saturating_add(I7F1::from_bits(40))
-            .saturating_to_num::<u8>()
-            .clamp(0, 0x50);
-
+        let gain_to_bits = |gain: I7F1| -> u8 {
+            let gain = gain.saturating_add(I7F1::from_bits(0x28))
+                .to_bits()
+                .clamp(0, 0x50);
+            unsafe { mem::transmute(gain) }
```

Comment:
transmute is used once, so you could just call it via: `core::mem::transmute(gain)` and drop the `use core::mem`.

Reply:
Done in c5f39d5c89240d17b0d7c8ee48fc48757c163bba & 07987aea4e2706d8a7fa252ba3f6e31d576537ee.